lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,9 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    attributes :values, :headers
+
+    def initialize(attrs = {})
+      @attributes = attrs
+    end
```

Comment:
I think, both variables will have the reference to one hash, it should not be copied in memory.